src/api/routes.ts

```diff
@@ -17,11 +17,11 @@ export const buildDeleteItemRoute = (id: UUID) => `${ITEMS_ROUTE}/${id}`;
 export const buildDeleteItemsRoute = (ids: UUID[]) =>
   `${ITEMS_ROUTE}?${qs.stringify({ id: ids }, { arrayFormat: 'repeat' })}`;
 export const buildGetChildrenRoute = (id: UUID) =>
-  `${ITEMS_ROUTE}/${id}/children`;
+  `${ITEMS_ROUTE}/${id}/children?ordered=true`;
 export const buildGetItemRoute = (id: UUID) => `${ITEMS_ROUTE}/${id}`;
 export const buildGetPublicItemRoute = (id: UUID) => `p/${ITEMS_ROUTE}/${id}`;
 export const buildGetPublicChildrenRoute = (id: UUID) =>
-  `p/${ITEMS_ROUTE}/${id}/children`;
+  `p/${ITEMS_ROUTE}/${id}/children?ordered=true`;
```

Comment:
If this can be done without too much changes in the other repositories, let's have an parameter in the function as well:
```buildGetChildrenRoute = (id, {ordered = true} = {}) => {...}```